src/components/accession2/properties/CollectedReceivedDate2.tsx

```diff
@@ -84,7 +86,8 @@ export default function CollectedReceivedDate2({ onChange, record, type, validat
           value={dates.collectedDate}
           onChange={(value) => changeDate('collectedDate', value)}
           errorText={dateErrors.collectedDate}
-          maxDate={Date.now()}
+          maxDate={timeZone ? changeTimezone(new Date(), timeZone) : new Date()}
```

Comment:
we can assume timeZone will always be set

Reply:
this can just be `maxDate={changeTimezone(new Date(), timeZone}`
we should rename changeTimezone to changeTimeZone since we use 'time zone' and not 'timezone'. can we do that? i missed it in the earlier review.